csharp/src/Ice/MetricsObserverI.cs

```diff
@@ -194,20 +194,32 @@ public string resolve(MetricsHelper<T> helper, string attribute)
             return resolver.resolveImpl(helper);
         }
 
-        public void
-        add(string name, System.Reflection.MethodInfo method) => _attributes.Add(name, new MethodResolverI(name, method));
-
-        public void
-        add(string name, System.Reflection.FieldInfo field) => _attributes.Add(name, new FieldResolverI(name, field));
-
-        public void
-        add(string name, System.Reflection.MethodInfo method, System.Reflection.FieldInfo field) => _attributes.Add(name, new MemberFieldResolverI(name, method, field));
-
-        public void add(string name, System.Reflection.MethodInfo method, System.Reflection.PropertyInfo property) =>
-            _attributes.Add(name, new MemberPropertyResolverI(name, method, property));
+        public void add(
+            string name,
+            System.Reflection.MethodInfo method) => _attributes.Add(name, new MethodResolverI(name, method));
+
+        public void add(
+            string name,
+            System.Reflection.FieldInfo field) => _attributes.Add(name, new FieldResolverI(name, field));
+
+        public void add(
+            string name,
+            System.Reflection.MethodInfo method,
+            System.Reflection.FieldInfo field) =>
+                _attributes.Add(name, new MemberFieldResolverI(name, method, field));
+
+        public void add(
+            string name,
+            System.Reflection.MethodInfo method,
+            System.Reflection.PropertyInfo property) =>
+                _attributes.Add(name, new MemberPropertyResolverI(name, method, property));
```

Comment:
No need to indent `_attributes` here.
```suggestion
      _attributes.Add(name, new MemberPropertyResolverI(name, method, property));
```